data/data/gcp/network/external-lb.tf

```diff
@@ -5,7 +5,7 @@ resource "google_compute_address" "cluster_public_ip" {
 resource "google_compute_http_health_check" "api_external" {
   name = "${var.cluster_id}-api-external"
 
-  port         = 6443
+  port         = 6080
```

Comment:
you need to make sure the firewalls are open for external health-checks from 6080
and internal health-checks from 6443